packages/plugin-npm/sources/NpmHttpFetcher.ts

```diff
@@ -50,13 +51,9 @@ export class NpmHttpFetcher implements Fetcher {
     if (archiveUrl === null)
       throw new Error(`Assertion failed: The archiveUrl querystring parameter should have been available`);
 
-    const sourceBuffer = await httpUtils.get(archiveUrl, {
+    return npmHttpUtils.get(archiveUrl, {
```

Comment:
I'm not sure if stylistically you would prefer:

```suggestion
    return await npmHttpUtils.get(archiveUrl, {
```